Formula/mad.rb

```diff
@@ -22,10 +23,25 @@ class Mad < Formula
     sha256 "7bd46d4da0f695b3a5bcc899b7139f14d11741f2e47d34f21a984f9bab953c81" => :mavericks
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
   def install
+    touch "NEWS"
+    touch "AUTHORS"
+    touch "ChangeLog"
```

Comment:
Why is that necessary?

Reply:
`autoreconf -fiv` would not succeed otherwise